### DISCUSSION TITLE
Add-on version 0.6

### DISCUSSION THREAD

**Discussion by hama3254** [hama3254/CRD-add-on#1] [Show and tell]
The new add-on version 0.6 is now available and supports the new funimation website and the crunchyroll beta website ( requires v3.8-Ealry-Beta) 

The Firefox version is available in the mozilla add-on gallery : https://addons.mozilla.org/de/firefox/addon/crunchyroll-downloader-addon/

The Chrome version can be found in the wiki article : https://github.com/hama3254/Crunchyroll-Downloader-v3.0/wiki/Chrome-add-on-installation

Old usage tutorial :  https://www.youtube.com/watch?v=ZHCH3rNWYS8

![grafik](https://user-images.githubusercontent.com/52826614/122682463-c3fafd80-d1f9-11eb-9fc2-631c7741d342.png)

**Comment by ghost** [comment]
Moin moin,

Edit Nvm.

Ich habe nur das Problem gerade, dass der Import ziemlich lange dauert und wenn es drin ist der Download nicht startet.
Version ist die aktuellste.

**Reply by hama3254** [reply]
Was heißt "lange" ? das sollte schlimmsten falls paar Sekunden dauern und der download ist in den meisten fällen in der warte schleife aber das sollte er auch anzeigen.

**Reply by ghost** [reply]
Der Import lag wohl ein einer alten Mozilla Version oder so. Hat sich von alleine gelöst.

Allerdings das mit dem Download nicht. Zwar macht er einige aber leider nicht alle:
https://streamable.com/mt8xqf

**Reply by hama3254** [reply]
bei dem video ist es eine premium folge, im browser bist du eingeloggt ist der account premium und bist du im downloader selbst eingeloggt?

**Reply by ghost** [reply]
Im Browser und im Downloader bin ich angemeldet. Hat auch ebenso Premium.

**Reply by hama3254** [reply]
bei genauem hinsehen fällt auf das nur das Hauptfenster die falsche anzeige behält vermutlich weil die erst später dazugekommen ist, er springt auf `Status: idle`  dabei müsste er eigentlich noch eine popup box anzeigen mit einem fehler. 
schau mal ob du eine `html.log` im ordner vom downloader findest.

**Reply by ghost** [reply]
Nicht vorhanden nein.

**Reply by hama3254** [reply]
Das müsste ich mir mal anschauen aber mein Account ist im beta program weshalb normale Premium befolgen nicht gehen weil es mich auf die Beta Version weiterleitet.

**Reply by ghost** [reply]
Wenne Hilfe brauchst sag Bescheid. Würde gerne helfen.
Wollte dich ohnehin noch wegen einer ev. Zusatzfunktion fragen :D.

**Reply by hama3254** [reply]
für irgenwelche speziellen sachen wäre discord besser geeigent als github : Hama#4155

**Reply by ghost** [reply]
Hab dir btw vor 3h ne Anfrage geschickt :D

**Comment by kanohn** [comment]
Hello,

I'm trying to use the add on, when open the add-on on the browser, it ask me to set the port, i tried 80 and 8080, and nothing happens.

I have tried with the application opened and closed, nothing happens.. what going wrong?

No matter if i try on chrome or mozilla, nothing happens..

**Reply by hama3254** [reply]
the add-on needs to be enabled in the settings by choosing a port from the drop down or you can enter a number between 0 and 65 535 as text if you don't click on the arrow but on the text. 
This setting requires a restart to be applyed.
this port needs then to be set inside the add-on then it should work.
![grafik](https://user-images.githubusercontent.com/52826614/126693324-45f9e16d-8a0a-4e24-b474-dcce2d3e27b2.png)

**Reply by kanohn** [reply]
Worked! Thanks!

I tried with ports 80 and 8080 and it seems not worked, but with other port worked fine.
